packages/serverless/test/gcpfunction.test.ts

```diff
@@ -19,20 +19,25 @@ import {
  * A hack-ish way to contain everything related to mocks in the same __mocks__ file.
  * Thanks to this, we don't have to do more magic than necessary. Just add and export desired method and assert on it.
  */
+type objectOfStrings = { [key: string]: string };
```

Comment:
Can we name this something slightly more descriptive? Even better, I'd probably just scrap the type entirely and just use `{ [key: string]: string }` or `Record<string, string>` everywhere you're using `objectOfStrings`. It's almost the same number of characters, and doesn't force the reader to translate.

(If you do keep the type, please remember to capitalize the first letter.)

Reply:
Good point! Thanks for your help, made the changes!